sulong/projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/interop/DynamicMutexTypeCastInteropTest.java

```diff
@@ -48,7 +48,7 @@
 import com.oracle.truffle.llvm.spi.NativeTypeLibrary;
 import com.oracle.truffle.llvm.test.interop.values.ArrayObject;
 
-public class DynamicTypeCastInteropTest extends InteropTestBase {
+public class DynamicMutexTypeCastInteropTest extends InteropTestBase {
```

Comment:
why has this been renamed?

Reply:
that was by accident, renamed it now back to original.